package.json

```diff
@@ -10,6 +10,8 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "date-fn": "^0.0.2",
```

Comment:
Was this installed by accident? If so i would uninstall it before merging the pull request

``` bash
npm uninstall date-fn
```

Reply:
yeees, it was an accident!